### PR TITLE
Send operation name to the server

### PR DESCRIPTION
**Pull request by Poincare**
Sending the operation name of the query down the server (probably helpful for logging, etc.).

TODO:

- [x] Update CHANGELOG.md with your change
- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass

### CODE REVIEW

File: src/networkInterface.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export interface Request {
   debugName?: string;
   query?: Document;
   variables?: Object;
+  operationName?: String;
```
Comment (helfer) [inline src/networkInterface.ts]: `getOperationName` returns `string`, but this definition says `String`. Should probably be the same throughout.

File: src/queries/getFromAST.ts [hunk 2]
```diff
@@ -44,6 +44,17 @@ string in a "gql" tag? http://docs.apollostack.com/apollo-client/core.html#gql`)
   }
 }
 
+export function getOperationName(doc: Document): string {
+  let res: string = '';
+  doc.definitions.forEach((definition) => {
```
Comment (helfer) [inline src/queries/getFromAST.ts]: technically you could stop once you've found the first definition.

**Comment by helfer** [comment]
lgtm, apart from the `string` vs `String` thing.

**Comment by helfer** [comment]
Actually, changelog entry seems to be missing. For some reason you deleted it again.

**Comment by Poincare** [comment]
Fixed.

**Pull request merged**